frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -65,17 +74,34 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
     <div className="ceph-ocs-install__chart-wrapper">
       <div className="ceph-ocs-install_capacity-header">Selected Capacity</div>
       <div className="ceph-ocs-install__stats">
-        <div>{pluralize(nodes.length, 'Node')}</div>
+        <div>
+          {state.filteredNodes.length ? (
+            <Button
+              component="a"
+              variant="link"
+              onClick={() => dispatch({ type: 'setShowNodeList', value: true })}
```

Comment:
```suggestion
              onClick={dispatch({ type: 'setShowNodeList', value: true })}
```

Reply:
This will give you  - `Type 'void' is not assignable to type '(event: MouseEvent<HTMLButtonElement, MouseEvent>) => void'.ts(2322)` esLint error.